packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -241,6 +241,9 @@ export const useUserButtonContext = () => {
   const afterSwitchSessionUrl = ctx.afterSwitchSessionUrl || displayConfig.afterSwitchSessionUrl;
   const navigateAfterSwitchSession = () => navigate(afterSwitchSessionUrl);
 
+  const userProfileMode =
+    !!ctx.userProfileUrl && ctx.userProfileMode === undefined ? 'navigation' : ctx.userProfileMode;
```

Comment:
Can we just do `!ctx.userProfileMode` here or do we care for `null` ?

Reply:
Yeah you are right we can change that to be `!ctx.userProfileMode`, I also don't think that this can be `null`